### DISCUSSION TITLE
Does KanbanBoard sample run in Release mode on iOS device?

### DISCUSSION THREAD

**Discussion by Ghevi** [VladislavAntonyuk/MauiSamples#80] [Q&A] [answered]
I ask this because I've encountered this kind of exception on my project when publishing the app to iOS device, here's the [issue](https://github.com/dotnet/maui/issues/9176)

**Comment by VladislavAntonyuk** [comment]
I can't answer you immediately. It definitely works in Debug mode. I will try in Release

**Reply by VladislavAntonyuk** [reply]
The KanbanBoard crashes in Release mode.
SQLite sample works fine.

**Comment by VladislavAntonyuk** [comment]
*** Terminating app due to uncaught exception 'System.AggregateException', reason: 'AggregateException_ctor_DefaultMessage (Attempting to JIT compile method '(wrapper dynamic-method) KanbanBoard.Models.Card object:Thunk1ret_Card_QueryContext_DbDataReader_ResultContext_SingleQueryResultCoordinator (System.Func`2<object[], object>,Microsoft.EntityFrameworkCore.Query.QueryContext,System.Data.Common.DbDataReader,Microsoft.EntityFrameworkCore.Query.Internal.ResultContext,Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryResultCoordinator)' while running in aot-only mode. See https://docs.microsoft.com/xamarin/ios/internals/limitations for more information.

**Comment by VladislavAntonyuk** [comment]
It should be fixed as part of .NET 9 release

**Comment by VladislavAntonyuk** [comment, marked as answer]
It works
![image](https://github.com/user-attachments/assets/2d806f98-f19c-418e-970d-7f5e29fec48f)
